src/main/java/org/owasp/esapi/logging/java/JavaLoggerLevel.java

```diff
@@ -0,0 +1,29 @@
+package org.owasp.esapi.logging.java;
+
+import java.util.logging.Level;
+
+/**
+ *  A custom logging level defined between Level.SEVERE and Level.WARNING in logger.
+ */
+public class JavaLoggerLevel extends Level {
```

Comment:
This should be renamed to something more obvious of the intent.

Reply:
This is the custom level map that ESAPI uses to record ERROR messages in the Java log context.

Reply:
Addressed in next push.  Now named ESAPIErrorJavaLevel